stable/graylog/templates/configmap.yaml

```diff
@@ -140,16 +140,25 @@ data:
     GRAYLOG_HOME=/usr/share/graylog
     MASTER_NAME="{{ template "graylog.fullname" . }}-master.{{ .Release.Namespace }}.svc.cluster.local"
     # Looking for Master IP
-    MASTER_IP=`/k8s/kubectl --namespace {{ .Release.Namespace }} get pod -o jsonpath='{range .items[*]}{.metadata.name} {.status.podIP}{"\n"}{end}' -l graylog-role=master --field-selector=status.phase=Running|awk '{print $2}'`
+    MASTER_IP=`/k8s/kubectl --namespace {{ .Release.Namespace }} get pod -o jsonpath='{range .items[*]}{.metadata.name} {.status.podIP}{"\n"}{end}' -l graylog-role=
+    --field-selector=status.phase=Running|awk '{print $2}'`
```

Comment:
`graylog-role=master` is missing and it should not be move to another line. It will cause command not found error.